cmd/node/node.go

```diff
@@ -319,6 +319,8 @@ func main() {
 	n := node.NewNode(services, declAddr, bindAddr, proto.NewTimestampFromUSeconds(outdatePeriod))
 	go n.Run(ctx, parent, services.InternalChannel)
 
+	go sched.Reschedule()
```

Comment:
What is the point of doing reschedule here? There is no connection and no consensus at this moment. I think this is pointless.

Reply:
Node can mining without connections. There should be initial schedule request.